src/tests/ftest/soak/soak.py

```diff
@@ -447,14 +502,21 @@ def setUp(self):
             raise SoakTestError(
                 "<<FAILED: Partition is not correctly setup for daos "
                 "slurm partition>>")
-
-        # include test node for log cleanup; remove from client list
-        self.test_node = [socket.gethostname().split('.', 1)[0]]
-        if self.test_node[0] in self.hostlist_clients:
-            self.hostlist_clients.remove(self.test_node[0])
-            self.log.info(
+        # Check if the server nodes are in the client list;
+        # this will happen when only one partition is specified
+        for host_server in self.hostlist_servers:
+            if host_server in self.hostlist_clients:
+                self.hostlist_clients.remove(host_server)
+        self.log.info(
                 "<<Updated hostlist_clients %s >>", self.hostlist_clients)
-        self.node_list = self.hostlist_clients + self.test_node
+        # # include test node for log cleanup; remove from client list
+        # self.test_node = [socket.gethostname().split('.', 1)[0]]
+        # if self.test_node[0] in self.hostlist_clients:
+        #     self.hostlist_clients.remove(self.test_node[0])
+        #     self.log.info(
+        #         "<<Updated hostlist_clients %s >>", self.hostlist_clients)
+        # self.node_list = self.hostlist_clients + self.test_node
```

Comment:
I left this code in and will uncomment when the test node is not included as a client node.
Should I remove it for now?